package.json

```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "git://github.com/verdaccio/ui"
   },
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
This is wrong, the main field is being used by node in runtime. This means node only can read JS and TS is only being used in development.